feature/qos/tests/qos_ecn_config_test/qos_ecn_config_test.go

```diff
@@ -471,12 +485,21 @@ func testCiscoECNConfig(t *testing.T) {
 	if got, want := gnmi.GetConfig(t, dut, wredUniform.EnableEcn().Config()), ecnConfig.ecnEnabled; got != want {
 		t.Errorf("wredUniform.EnableEcn().State(): got %v, want %v", got, want)
 	}
+	if deviations.EcnSameMinMaxThresholdUnsupported(dut) {
```

Comment:
Can we create two variables here for wantMinThreshold and wantMaxThreshold and apply the deviation logic to those? We need not copy the whole if/else statements

Reply:
modified